source/Activated.js

```diff
@@ -1,5 +1,6 @@
-import React, { useCallback, useEffect, useRef, useState } from 'react'
 import { bool, func, node, object, string } from 'prop-types'
+import React, { useCallback, useEffect, useRef, useState } from 'react'
```

Comment:
FYI -- this is annoying, but I think we always want to keep `react` as the first import no matter what. There was something back in the day recommending this b/c of different versions of react being run if a dependency also requires react...

Reply:
once you move this to the top, merge away